package.json

```diff
@@ -128,6 +128,7 @@
     "node-sass": "4.12.0",
     "prettier": "1.16.4",
     "pretty-quick": "1.10.0",
+    "prop-types": "^15.6.1",
```

Comment:
Is this required? If so, could you please lock the version?

Reply:
Not exactly required, but it's removing some warning about peer dependencies during builds that we had for a long time.
Yes i'm going to lock the version.

Reply:
done